subsys/bluetooth/controller/ll_sw/ull_llcp_hci.c

```diff
@@ -117,13 +117,12 @@ uint8_t ll_apto_set(uint16_t handle, uint16_t apto)
 		return BT_HCI_ERR_UNKNOWN_CONN_ID;
 	}
 
-	return BT_HCI_ERR_CMD_DISALLOWED;
+	// apto unit is 10ms, RADIO_CONN_EVENTS converts us to connection events
+	conn->apto_reload = RADIO_CONN_EVENTS(apto * 10U * 1000U,
+					      conn->lll.interval * CONN_INT_UNIT_US);
+
+	return 0;
```

Comment:
This will fail test_hci_apto in tests/bluetooth/controller/ctrl_hci/bluetooth.controller.ctrl_hci.test

Reply:
fixed